elt.js

```diff
@@ -34,9 +62,9 @@ window.ELT = {
 
 	api: {
 		participant: createApiActionFor('participants/{}'),
-		participantDonations: createApiActionFor('participants/{}/donations'),
+		participantDonations: createDateApiActionFor('participants/{}/donations?where=createdDateUTC>[]'),
```

Comment:
Won't this break the `donation-rotation` widget?  It wants all donations, not just the newest ones, and also uses this API call.

Reply:
Fixed